build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
 
-        spineVersion = '0.7.16-SNAPSHOT'
+        spineVersion = '0.7.18-SNAPSHOT'
```

Comment:
I needed to advance in two version numbers to solve dependency issues from Projects.